services/motion/builtin/builtin_test.go

```diff
@@ -1315,13 +748,28 @@ func TestReplanning(t *testing.T) {
 		injectedMovementSensor, _, kb, ms := createMoveOnGlobeEnvironment(ctx, t, gpsPoint, spatialmath.NewPoseFromPoint(tc.noise), 5)
 		defer ms.Close(ctx)
 
-		success, err := ms.MoveOnGlobe(ctx, kb.Name(), dst, 0, injectedMovementSensor.Name(), nil, motionCfg, tc.extra)
+		req := motion.MoveOnGlobeReq{
+			ComponentName:      kb.Name(),
+			Destination:        dst,
+			Heading:            0,
+			MovementSensorName: injectedMovementSensor.Name(),
+			MotionCfg:          motionCfg,
+			Extra:              tc.extra,
+		}
+		executionID, err := ms.MoveOnGlobe(ctx, req)
+		test.That(t, err, test.ShouldBeNil)
+
+		timeoutCtx, timeoutFn := context.WithTimeout(ctx, time.Minute*5)
```

Comment:
These tests take an absurd amount of time (2 min on my machine). We really need to speed these tests up, they shouldn't take this long.

Reply:
I think it gets better if we make the distance to plan smaller, as I think its quite long at the moment